src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/Offender.java

```diff
@@ -166,6 +166,11 @@ public class Offender extends AuditableEntity {
     @Column(name = "LAST_NAME_ALPHA_KEY")
     private String lastNameAlphaKey;
 
+    @OneToMany(mappedBy = "offender", cascade = CascadeType.ALL, fetch = FetchType.LAZY)
+    @Default
+    @Exclude
+    private List<OffenderAddress> addresses = new ArrayList<>();
```

Comment:
I take it Prison API will return an empty list rather than `null` if there are no addresses?